shared/osarch/architectures.go

```diff
@@ -73,10 +73,10 @@ var architectureSupportedPersonalities = map[int][]int{
 	ARCH_64BIT_POWERPC_BIG_ENDIAN:    {ARCH_32BIT_POWERPC_BIG_ENDIAN},
 	ARCH_64BIT_POWERPC_LITTLE_ENDIAN: {},
 	ARCH_64BIT_S390_BIG_ENDIAN:       {},
-	ARCH_32BIT_MIPS_BIG_ENDIAN:       {},
-	ARCH_32BIT_MIPS_LITTLE_ENDIAN:    {},
-	ARCH_64BIT_MIPS64_BIG_ENDIAN:     {ARCH_32BIT_MIPS_BIG_ENDIAN},
-	ARCH_64BIT_MIPS64_LITTLE_ENDIAN:  {ARCH_32BIT_MIPS_LITTLE_ENDIAN},
+	ARCH_32BIT_MIPS:                  {},
+	ARCH_64BIT_MIPS:                  {ARCH_32BIT_MIPS},
+	ARCH_32BIT_RISCV_LITTLE_ENDIAN:   {},
+	ARCH_64BIT_RISCV_LITTLE_ENDIAN:   {},
```

Comment:
Presumably should have `ARCH_32BIT_RISCV_LITTLE_ENDIAN` as a personality?

Reply:
Theres no ability to run 32bit code natively on a 64 bit core - they're slightly different ISAs and there would need to be some emulation to allow that to happen so I didn't add the personalities as it would probably lead to some confusion when it doesn't work.

Reply:
Ok, good to know.